### PR TITLE
[prometheus-memcached-exporter] Fix: allow to configure extra labels …

### PR DESCRIPTION
**Pull request by rsicart**
…and annotations for the Service

#### Which issue this PR fixes

*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*

- fixes #

#### Checklist

- [x] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

**Comment by rsicart** [comment]
Some maintainer or admin could review plz? I'm the only maintainer of this chart and I can't validate myself :D

File: charts/prometheus-memcached-exporter/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: prometheus-memcached-exporter
 description: Prometheus exporter for Memcached metrics
 type: application
-version: 0.1.0
+version: 0.1.1
```
Comment (zeritti) [inline charts/prometheus-memcached-exporter/Chart.yaml:5]: Adding new features, e.g. new fields, needs bumping the minor version number.

```suggestion
version: 0.2.0
```

File: charts/prometheus-memcached-exporter/values.yaml [hunk 2]
```diff
@@ -39,6 +39,10 @@ securityContext: {}
 service:
   type: ClusterIP
   port: 9150
+  # Labels to append to the service
+  extraLabels: {}
```
Comment (zeritti) [inline charts/prometheus-memcached-exporter/values.yaml:43]: It is quite common in the repository's charts to name the field in this context just `labels`. Would it be possible to rename `extraLabels`?

**Review by zeritti — CHANGES_REQUESTED** [review]
Thank you, @rsicart, for your PR. Please, see my comments below.

File: charts/prometheus-memcached-exporter/Chart.yaml (hunk 1 above)
Comment (rsicart) [reply charts/prometheus-memcached-exporter/Chart.yaml:5]: thanks a lot for reviewing! I'll change that.

File: charts/prometheus-memcached-exporter/values.yaml (hunk 2 above)
Comment (rsicart) [reply charts/prometheus-memcached-exporter/values.yaml:43]: sure thing :)

**Review by zeritti — APPROVED** [review]
Thank you, LGTM.

**Comment by rsicart** [comment]
Thanks a lot @zeritti !

**Pull request merged**